gatsby-config.js

```diff
@@ -29,7 +29,6 @@ module.exports = {
         apiURL: 'http://techconnect-api.ddns.net:1337',
         contentTypes: [ // List of the Content Types you want to be able to request from Gatsby.
           'project',
```

Comment:
Comma will create merge conflicts
```suggestion
          'project'
```